js/webpack_get_defaults.js

```diff
@@ -219,17 +236,11 @@ module.exports = function (basePath) {
         },
         {
           test: /\.css/,
-          use: extractCSS.extract({
-            loader: [cssLoader, postCssLoader],
-            fallback: 'style-loader',
-          }),
+          use: cssRules([cssLoader, postCssLoader], extractCSS)
```

Comment:
Maybe be more explicit about using or not `extractText` ?
enableExtractText() ? useExtractText(loaders) : useStyleLoader(loaders)